schemas/Error.json

```diff
@@ -52,6 +52,26 @@
         {"const": "Native", "description": "Native / legacy crypto module specific to each platform."},
         {"const": "Rust", "description": "Shared / cross-platform crypto module written in Rust."}
       ]
+    },
+    "eventLocalAge": {
+      "description": "An heuristic based on event origin_server_ts and the current device creation time (origin_server_ts - device_ts). This would be used to get the source of the event scroll-back/live/initialSync.",
+      "type": "integer"
```

Comment:
Maybe add the unit in the name and description? `eventLocalAgeMillis` or something similar...

I know that `origin_server_ts` is millis, but just to avoid any confusion...